src/main/java/com/hillel/dzhurynskyi/homework/hm13/ValueCalculator.java

```diff
@@ -0,0 +1,63 @@
+package com.hillel.dzhurynskyi.homework.hm13;
+
+public class ValueCalculator {
+    private static final int size = 10000000;
+    private final int halfSize = size / 2;
+    private float[] array = new float[size];
+
+    public void doSomethingWithMultithreading() {
+        long start = System.currentTimeMillis();
+
+        fillArray();
+
+        float[] a1 = new float[halfSize];
+        float[] a2 = new float[halfSize];
+
+        System.arraycopy(array, 0, a1, 0, halfSize);
+        System.arraycopy(array, halfSize, a2, 0, halfSize);
+
+        Thread thread1 = new Thread(() -> operationWithArray(a1));
+        Thread thread2 = new Thread(() -> operationWithArray(a2));
+        thread1.start();
+        thread2.start();
+
+        try {
+            thread2.join();
+        } catch (InterruptedException e) {
+            throw new RuntimeException(e);
+        }
+
+        System.arraycopy(a1, 0, array, 0, halfSize);
```

Comment:
You have additional operations between multithreading/one_thead apps - `arraycopy`. Without it, the difference in time will be more sensitive